### DISCUSSION TITLE
Cmdlet "Move-PnPFile -IgnoreVersionHistory" is not working as expected.

### DISCUSSION THREAD

**Discussion by 4iJacob** [pnp/powershell#3104] [Q&A]
HI All,

Following this issue which should be fixed I must be doing something wrong: [link](https://github.com/pnp/powershell/issues/1376#issuecomment-1496591214)

Cmdlet "Move-PnPFile -IgnoreVersionHistory" is not working as expected.

Expected behavior
When using "Move-PnPFile -IgnoreVersionHistory" i expect to see in the target site only the last version for the file moved.

Actual behavior
When using "Move-PnPFile -IgnoreVersionHistory" it moves all version history for one file.

Steps to reproduce behavior
$job = Move-PnPFile -SourceUrl $SourceFileURL -TargetUrl $TargetFileURL -IgnoreVersionHistory -AllowSchemaMismatch  -NoWait  

What is the version of the Cmdlet module you are running?
Using PnP.PowerShell 2.1.1 and PowerShell 7.3.4

**Comment by 4iJacob** [comment]
What am I missing here, is it working for you?

**Reply by AndersRask** [reply]
Yeah when i test it i see the same issue.
Even when specifying IgnoreVersionHistory the version history is included

@veronicageek this still doesnt seem to work in 2.1.1
https://github.com/pnp/powershell/issues/1376

**Comment by 4iJacob** [comment]
Can [this ](https://github.com/pnp/powershell/issues/1376)be reopened again @veronicageek

**Comment by veronicageek** [comment]
@4iJacob - Please open a new issue in the repo **filling the template**. Thanks

**Comment by Micael-stack** [comment]
Hi all, its old and still not fixed. But its not from Pnp side, its Ms CopyMigration API that is broken. The param is ignored by their API and still is in 2025. I Guess they rather have AI somewhere than fixing their thing.
